src/collapse/docs/readme.md

```diff
@@ -7,3 +7,27 @@
   <i class="glyphicon glyphicon-eye-open"></i>
   _(Default: `false`)_ -
   Whether the element should be collapsed or not.
+
```

Comment:
Hmm, looks like `uib-alert` documentation is the exception.  How does the following look?
```md
* `collapsing`
  <small class="badge">$</small> -
  An optional expression called before the element begins collapsing.
  If the expression returns a promise, animation won't start until the promise resolves.
  If the returned promise is rejected, collapsing will be cancelled.
  
* `collapsed`
  <small class="badge">$</small> -
  An optional expression called after the element finished collapsing.
  
* `expanding`
  <small class="badge">$</small> -
  An optional expression called before the element begins expanding.
  If the expression returns a promise, animation won't start until the promise resolves.
  If the returned promise is rejected, expanding will be cancelled.
  
* `expanded`
  <small class="badge">$</small> -
  An optional expression called after the element finished expanding.
```